docs/testing/integration-tests.md

```diff
@@ -55,7 +55,7 @@ near-sdk = { path = "../../near-sdk" }
 near-units = "0.2.0"
 serde_json = "1.0"
 tokio = { version = "1.14", features = ["full"] }
-workspaces = "0.3.1"
+workspaces = "0.4.0"
```

Comment:
to avoid issues while moving content to docs.near.org, I'd commit @austinabell 's suggestion now, and we can make further changes later once the articles have been migrated.